mods/ts/rules/defaults.yaml

```diff
@@ -41,19 +51,34 @@
 		TerrainTypes: Clear, Road, DirtRoad, Rough
 		Adjacent: 4
 	FrozenUnderFog:
-	GivesBuildableArea:
-	Capturable:
 	SoundOnDamageTransition:
 		DamagedSounds: expnew01.aud
 		DestroyedSounds: crmble2.aud
 	RenderBuilding:
 	WithBuildingExplosion:
+	EngineerRepairable:
+	ShakeOnDeath:
+	ProximityCaptor:
```

Comment:
That one isn't needed in any of the official mods. Some Capture the Flag maps use it, but they can define it for themselves.